recipes/_compute_base_config.rb

```diff
@@ -47,6 +47,14 @@
   action %i[mount enable]
 end
 
+# Mount /opt/intel over NFS
+mount '/opt/intel' do
+  device "#{nfs_master}:/opt/intel"
+  fstype 'nfs'
+  options 'hard,intr,noatime,vers=4,_netdev'
```

Comment:
so far we've been using nfs version 3 everywhere. I agree we could update if there are no drawbacks but I would rather do it for all the mount points (I recall there were some performance degradations with version 4 we want to double check)

Reply:
Can you send me a link to that performance issue? I don't have any context for that.